R/format_PET.R

```diff
@@ -0,0 +1,1200 @@
+                .data$CaptureAlive, .data$ReleaseAlive,
+                .data$CapturePopID, .data$CapturePlot,
+                .data$ReleasePopID, .data$ReleasePlot,
+                .data$Mass, .data$Tarsus, .data$OriginalTarsusMethod,
+                .data$WingLength, .data$Age_observed, .data$Age_calculated,
+                .data$ChickAge, .data$ExperimentID,
+                #### mantain also BroodID needed to create Individual_data, remove at final step
+                .data$BroodID)
+
+
+  Capture_data_parmaj <-
+    bind_rows(Capture_data_parmaj_adults,
+              Capture_data_parmaj_chicks) %>%
+    #### Create new variables
+    dplyr::group_by(.data$IndvID) %>%
+    dplyr::arrange(.data$BreedingSeason, .data$CaptureDate) %>% ## Use .data$CaptureDate if the data owner provides one. Now use StartDateOfLaying1May1.
+    dplyr::mutate(CaptureID = paste(.data$IndvID, row_number(), sep = "_")) %>%
+    dplyr::ungroup() %>%
+    #### Final arrangement
+    dplyr::select(.data$CaptureID, .data$IndvID, .data$Species,
+                  .data$Sex_observed, .data$BreedingSeason,
+                  .data$CaptureDate, .data$CaptureTime,
+                  .data$ObserverID, .data$LocationID,
+                  .data$CaptureAlive, .data$ReleaseAlive,
+                  .data$CapturePopID, .data$CapturePlot,
+                  .data$ReleasePopID, .data$ReleasePlot,
+                  .data$Mass, .data$Tarsus, .data$OriginalTarsusMethod,
+                  .data$WingLength, .data$Age_observed, .data$Age_calculated,
+                  .data$ChickAge, .data$ExperimentID,
+                  #### mantain also BroodID needed to create Individual_data, remove at final step
+                  .data$BroodID)
+
+
+  #### FICHYP
+  #### ----------------------
+
+  #### solve rings "бк" ring number series change >> Data owner
+
+  #### This ring is probably wrong?? is too long VT76392096 >> Data owner
+
+  Capture_data_fichyp_chicks_temp <-
+    fichyp_data %>%
+    dplyr::select(.data$Species, .data$PopID, .data$BreedingSeason,
+                  .data$BroodID, .data$LocationID,
+                  .data$TheLineOfArtificialNestBoxes,
+                  .data$StartDateOfLaying1May1,
+                  .data$NestlingRings) %>%
+    dplyr::filter(!is.na(.data$NestlingRings)) %>%
+
+    #### SOLVE THIS, TEMP REMOVE
+    dplyr::filter(NestlingRings != "бк") %>%
+    dplyr::filter(NestlingRings != "VT76-260-63") %>%
+
+    #### Separate series and ring codes
+    tidyr::separate(col = .data$NestlingRings, into = c("rings1", "rings2"),
+                    sep = ";", remove = FALSE) %>%
+    tidyr::separate(col = .data$rings1, into = c("rings1_start", "rings1_end"),
+                    sep = "-", remove = FALSE, convert = TRUE) %>%
+    dplyr::mutate(series1 = str_sub(rings1_start, 1, nchar(rings1_start)-nchar(rings1_end)),
+                  rings1_start = str_sub(rings1_start, nchar(series1)+1, nchar(rings1_start))) %>%
```

Comment:
Make sure variables in dplyr pipe are given explicit namespace with `.data$`. This flags as a NOTE in `devtools::check()`. Try to run this and look at the guidelines here https://github.com/SPI-Birds/pipelines#devtoolscheck
```suggestion
    dplyr::mutate(series1 = str_sub(.data$rings1_start, 1, nchar(.data$rings1_start)-nchar(.data$rings1_end)),
                  rings1_start = str_sub(.data$rings1_start, nchar(.data$series1)+1, nchar(.data$rings1_start))) %>%
```